commit 014135b89d23f15464d8a82f462ed0f9f98344e0
Author: Jan Steffens <jan.steffens@gmail.com>
Date:   Fri Sep 30 21:09:25 2011 +0200

    Revert "Sort alphabetically introspected files"

    This reverts commit 48998822d5e9575af822c1936b35be514dc2401b.

    Conflicts:

	telepathy-glib/introspection.am

diff --git a/telepathy-glib/introspection.am b/telepathy-glib/introspection.am
index 2027c5f..0c867cc 100644
--- telepathy-glib/introspection.am
+++ telepathy-glib/introspection.am
@@ -8,63 +8,64 @@ introspection_extra_dist = \
 if HAVE_INTROSPECTION

 INTROSPECTION_FILES = \
+	$(srcdir)/proxy.c _gen/proxy-introspectable.h \
	$(srcdir)/account.c $(srcdir)/account.h \
	$(srcdir)/account-channel-request.c $(srcdir)/account-channel-request.h \
	$(srcdir)/account-manager.c $(srcdir)/account-manager.h \
-	$(srcdir)/add-dispatch-operation-context.c $(srcdir)/add-dispatch-operation-context.h \
	$(srcdir)/automatic-client-factory.c $(srcdir)/automatic-client-factory.h \
	$(srcdir)/automatic-proxy-factory.c $(srcdir)/automatic-proxy-factory.h \
-	$(srcdir)/base-client.c $(srcdir)/base-client.h \
-	$(srcdir)/base-connection.c $(srcdir)/base-connection.h \
	$(srcdir)/basic-proxy-factory.c $(srcdir)/basic-proxy-factory.h \
-	$(srcdir)/capabilities.c $(srcdir)/capabilities.h \
-	$(srcdir)/channel.c $(srcdir)/channel.h \
-	$(srcdir)/channel-contacts.c \
-	$(srcdir)/channel-dispatcher.c $(srcdir)/channel-dispatcher.h \
-	$(srcdir)/channel-dispatch-operation.c $(srcdir)/channel-dispatch-operation.h \
-	$(srcdir)/channel-group.c \
-	$(srcdir)/channel-request.c $(srcdir)/channel-request.h \
	$(srcdir)/client-channel-factory.c $(srcdir)/client-channel-factory.h \
-	$(srcdir)/client-message.c $(srcdir)/client-message.h \
	$(srcdir)/connection.c $(srcdir)/connection.h \
-	$(srcdir)/connection-avatars.c \
	$(srcdir)/connection-contact-list.c $(srcdir)/connection-contact-list.h \
	$(srcdir)/connection-handles.c $(srcdir)/connection-contact-info.c \
+	$(srcdir)/connection-avatars.c \
+	$(srcdir)/protocol.c $(srcdir)/protocol.h \
	$(srcdir)/connection-manager.c $(srcdir)/connection-manager.h \
+	$(srcdir)/channel.c $(srcdir)/channel.h \
+	$(srcdir)/channel-contacts.c \
+	$(srcdir)/channel-group.c \
+	$(srcdir)/handle.c $(srcdir)/handle.h \
+	$(srcdir)/handle-channels-context.c $(srcdir)/handle-channels-context.h \
+	$(srcdir)/dbus-daemon.c $(srcdir)/dbus-daemon.h \
+	$(srcdir)/interfaces.c \
+	$(srcdir)/intset.c $(srcdir)/intset.h \
+	$(srcdir)/dbus.c $(srcdir)/dbus.h \
+	$(srcdir)/capabilities.c $(srcdir)/capabilities.h \
	$(srcdir)/contact.c $(srcdir)/contact.h \
	$(srcdir)/contact-operations.c $(srcdir)/contact-operations.h \
	$(srcdir)/contact-search.c $(srcdir)/contact-search.h \
	$(srcdir)/contact-search-result.c $(srcdir)/contact-search-result.h \
-	$(srcdir)/contacts-mixin.c $(srcdir)/contacts-mixin.h \
-	$(srcdir)/dbus.c $(srcdir)/dbus.h \
-	$(srcdir)/dbus-daemon.c $(srcdir)/dbus-daemon.h \
-	$(srcdir)/dbus-properties-mixin.c $(srcdir)/dbus-properties-mixin.h \
-	$(srcdir)/dbus-tube-channel.c $(srcdir)/dbus-tube-channel.h \
-	$(srcdir)/debug.c $(srcdir)/debug.h \
	$(srcdir)/defs.h \
-	$(srcdir)/errors.c $(srcdir)/errors.h \
-	$(srcdir)/file-transfer-channel.c $(srcdir)/file-transfer-channel.h \
-	$(srcdir)/group-mixin.c $(srcdir)/group-mixin.h \
-	$(srcdir)/handle.c $(srcdir)/handle.h \
-	$(srcdir)/handle-channels-context.c $(srcdir)/handle-channels-context.h \
-	$(srcdir)/handle-repo.c $(srcdir)/handle-repo.h $(srcdir)/handle-set.c \
-	$(srcdir)/interfaces.c \
-	$(srcdir)/intset.c $(srcdir)/intset.h \
-	$(srcdir)/message.c $(srcdir)/message.h \
-	$(srcdir)/observe-channels-context.c $(srcdir)/observe-channels-context.h \
-	$(srcdir)/presence-mixin.c $(srcdir)/presence-mixin.h \
-	$(srcdir)/protocol.c $(srcdir)/protocol.h \
-	$(srcdir)/proxy.c _gen/proxy-introspectable.h \
-	$(srcdir)/signalled-message.c $(srcdir)/signalled-message.h \
+	$(srcdir)/debug.c $(srcdir)/debug.h \
+	$(srcdir)/base-client.c $(srcdir)/base-client.h \
	$(srcdir)/simple-approver.c $(srcdir)/simple-approver.h \
	$(srcdir)/simple-client-factory.c $(srcdir)/simple-client-factory.h \
	$(srcdir)/simple-handler.c $(srcdir)/simple-handler.h \
	$(srcdir)/simple-observer.c $(srcdir)/simple-observer.h \
+	$(srcdir)/dbus-properties-mixin.c $(srcdir)/dbus-properties-mixin.h \
+	$(srcdir)/contacts-mixin.c $(srcdir)/contacts-mixin.h \
+	$(srcdir)/group-mixin.c $(srcdir)/group-mixin.h \
+	$(srcdir)/presence-mixin.c $(srcdir)/presence-mixin.h \
+	$(srcdir)/channel-dispatch-operation.c $(srcdir)/channel-dispatch-operation.h \
+	$(srcdir)/channel-request.c $(srcdir)/channel-request.h \
+	$(srcdir)/observe-channels-context.c $(srcdir)/observe-channels-context.h \
+	$(srcdir)/add-dispatch-operation-context.c $(srcdir)/add-dispatch-operation-context.h \
+	$(srcdir)/protocol.c $(srcdir)/protocol.h \
+	$(srcdir)/base-connection.c $(srcdir)/base-connection.h \
+	$(srcdir)/handle-repo.c $(srcdir)/handle-repo.h $(srcdir)/handle-set.c \
	$(srcdir)/stream-tube-channel.c $(srcdir)/stream-tube-channel.h \
+	$(srcdir)/dbus-tube-channel.c $(srcdir)/dbus-tube-channel.h \
	$(srcdir)/stream-tube-connection.c $(srcdir)/stream-tube-connection.h \
	$(srcdir)/text-channel.c $(srcdir)/text-channel.h \
+	$(srcdir)/file-transfer-channel.c $(srcdir)/file-transfer-channel.h \
+	$(srcdir)/client-message.c $(srcdir)/client-message.h \
+	$(srcdir)/message.c $(srcdir)/message.h \
+	$(srcdir)/signalled-message.c $(srcdir)/signalled-message.h \
+	$(srcdir)/channel-dispatcher.c $(srcdir)/channel-dispatcher.h \
+	$(srcdir)/errors.c $(srcdir)/errors.h \
	_gen/telepathy-enums.h _gen/telepathy-enums-gtk-doc.h \
-	_gen/telepathy-interfaces.h _gen/telepathy-interfaces-gtk-doc.h\
+	_gen/telepathy-interfaces.h \
	$(NULL)

 _gen/proxy-introspectable.h: proxy.h introspection.am
@@ -76,7 +77,6 @@ TelepathyGLib-0.12.gir: $(INTROSPECTION_SCANNER) \
		$(srcdir)/kludge.xsl \
		$(INTROSPECTION_FILES)
	$(AM_V_GEN)$(INTROSPECTION_SCANNER) -v \
-		--warn-all \
		--namespace TelepathyGLib \
		--nsversion=0.12 \
		--identifier-prefix=Tp \
